lib/smart_answer_flows/plan-adoption-leave.rb

```diff
@@ -29,15 +29,16 @@ def define
       date_question :leave_start? do
         calculate :start_date do |response|
           dist = (arrival_date - response).to_i
-          raise InvalidResponse unless (1..14).include? dist
+          raise InvalidResponse unless (1..14).cover? dist
           response
         end
 
         calculate :calculator do
           Calculators::PlanAdoptionLeave.new(
             match_date: match_date,
             arrival_date: arrival_date,
-            start_date: start_date)
+            start_date: start_date
+)
```

Comment:
I'm guessing this is a automatic fix. I sometimes 'correct' this, as while Rubocop doesn't have an issue with it, it still seems a bit odd to me. But I don't really mind either way.